### PR TITLE
Layout incident to alert

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://jira-hq.paloaltonetworks.local/browse/CRTX-47907
test run: https://code.pan.run/xsoar/content/-/pipelines/4696573

### CODE REVIEW

**Review by ShahafBenYakir — CHANGES_REQUESTED** [review]
DO NOT MERGE BWFORE SYNCING WITH @AradCarmi

File: demisto_sdk/commands/content_graph/objects/widget.py
Comment (AradCarmi) [inline demisto_sdk/commands/content_graph/objects/widget.py:25]:
```suggestion
            data = fix_widget_incident_to_alert(data)
```
```suggestion
            data = fix_incident_to_alert(data)
```

**Review by AradCarmi — DISMISSED** [review]
Look great!

File: demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py
Comment (dantavori) [inline demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py:14]: I saw that in some cases there is no `id` key with this value, but there is a `type` key (see [example](https://github.com/demisto/content/blob/master/Packs/CommonTypes/Layouts/layoutscontainer-Campaign.json#L55)) - is it by design to avoid these?

File: demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py
Comment (dantavori) [inline demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py:34]: without returning a value from `fix_recursively()` (just changing the relevant fields if needed) you won't need to have this check

**Review by dantavori — COMMENTED** [review]
looks great, I will approve after resolving my following comments

File: demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py
Comment (dorschw) [reply demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py:34]: True. Changing the relevant fields was the first attempt, but caused a headache with _modifying values while iterating on them_ (according to the interpreter at least, I disagreed), so I decided this comment is a price I'm willing to pay.

File: demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py
Comment (dorschw) [reply demisto_sdk/commands/prepare_content/preparers/replace_incident_alert.py]: Checked with @Ni-Knight, answer is positive, Implemented accordingly, please re-review :)

**Review by dantavori — APPROVED** [review]

**Review by ilaner — APPROVED** [review]
Impressive

**Comment by dorschw** [comment]
Will be merged as part of #3025

**Pull request closed without merging**
